solarforecastarbiter/reports/templates/html/head.html

```diff
@@ -5,4 +5,6 @@
 {# plotly js and the python plotly library do not have matching versions #}
 {% include "insert_plots.html" %}
 <script src="https://cdn.plot.ly/plotly-1.52.3.min.js"></script>
+<script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
```

Comment:
@lboeman do you have a non-jquery solution for this?

Reply:
fyi jquery is loaded already by the dashboard, but not in the html download. Could just not collapse the cost params if this is a problem

Reply:
nevermind then